outputs.tf

```diff
@@ -1,24 +1,26 @@
-output "arn" {
-  description = "The EC2 instance ARN."
-  value       = aws_instance.example.arn
+output "documentdb_arn" {
+  description = "The ARN of the DocumentDB cluster."
+  value       = module.documentdb-cluster.arn
```

Comment:
Why do you output only ARNs or other attributes for some of these but output entire objects for others (e.g., the `ec2` output)?

Reply:
Excellent question.  Since the admin password is marked as "sensitive" in the documentdb cluster module, if I add an output for the entire cluster, I get this error:

```
│ Error: Output refers to sensitive values
│ 
│   on outputs.tf line 2:
│    2: output "documentdb" {
│ 
│ To reduce the risk of accidentally exporting sensitive data that was intended to be only internal, Terraform requires that any root module output containing sensitive data be explicitly marked as sensitive, to confirm your intent.
│ 
│ If you do intend to export this data, annotate the output value as sensitive by adding the following argument:
│     sensitive = true
```

And when I add `sensitive = true` to the output, the output is displayed like this, which is not particularly useful:
```
documentdb = <sensitive>
```

If you know of a way to have it output the entire object and redact only the actual sensitive value (the admin password), I'm all ears.

Reply:
OK, can you add a comment to the variables where this sort of situation applies?

Reply:
Please see https://github.com/cisagov/cyhy-tf-root/pull/25/commits/e2ec4db45916b3c23ac99e8b083f82c3209c2ff4 and let me know if that works.